packages/components/asset/README.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Asset'
 slug: /components/asset/
 github: 'https://github.com/contentful/forma-36/tree/main/packages/components/asset'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-asset--basic'
+storybook: 'https://f36-storybook.netlify.app/?path=/story/components-asset--basic'
```

Comment:
They all need to be `https://....contentful.com` instead of `https://....netlify.app`